contrib/logging/fluentd-sidecar-gcp/config_generator.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mkdir -p /etc/google-fluentd/files
+if [ -z "$FILES_TO_COLLECT" ]; then
+  exit 0
+fi
+
+for filepath in $FILES_TO_COLLECT
+do
+  filename=$(basename $filepath)
+  cat > "/etc/google-fluentd/files/${filename}" << EndOfMessage
+<source>
+  type tail
+  format none
```

Comment:
It's a shame that if any files are in JSON format then any extra useful information won't get propagated. But for this I think this is fine.